<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html>

<head>
  <meta http-equiv="content-type" content="text/html; charset=utf-8" />
<style type="text/css"></style>
<meta http-equiv="Content-Type" content="text/html; charset=gb2312"/> 
        <style type="text/css">         
               @page {margin:1in 1in 0.5in 1in}

                .style1
                {
                        font-family:Arial;
                        font-size:13px;
                }

pre {
 white-space: pre-wrap;       /* css-3 */
 white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
 white-space: -pre-wrap;      /* Opera 4-6 */
 white-space: -o-pre-wrap;    /* Opera 7 */
 white-space: pre-line;
 word-wrap: break-word;       /* Internet Explorer 5.5+ */
}

        </style>
</head><body>

<table border="0" style="width:600px; BORDER-COLLAPSE: collapse;">
  <tr>
    <td align="center" valign="middle"><img src="https://ebridge.xjtlu.edu.cn/jiaotong_files/logo.gif"/>
    </td>
  </tr>
</table>

<p > <b>MODULE SPECIFICATION</b></p>
<br/>

		<table border="1" style="width: 600px; BORDER-COLLAPSE: collapse;">
            <tr>
                <td width=50% bgcolor="#ADD8E6"><b>Module Title</b> </td>
                <td width=50%>High-Throughput Approaches and Systems Biology </td>
            </tr>
            <tr>
                <td bgcolor="#ADD8E6"><b>Module Code</b></td>	
                <td>BIO316</td>
            </tr>
            <tr>
                <td bgcolor="#ADD8E6"><b>Originating Department/School</b></td>
                <td>Department of Biological Sciences</td>
            </tr>
            <tr>
                <td bgcolor="#ADD8E6"> <b>Module Level<sup>1</sup></b> </td>
                <td> 3  ( FHEQ level 6 )</td>
            </tr>
            <tr> 
                <td bgcolor="#ADD8E6"><b>Module Credits</b>&nbsp;<em>(normally 5)</em></td>
                <td>5</td>
            </tr>
            <tr> 
                <td bgcolor="#ADD8E6"><b>Pre-requisites</b>&nbsp;<em>(including Year 1)</em></td>
                 <td></td>
            </tr>
            <tr> 
                <td bgcolor="#ADD8E6"><b>Shared Programme(s)</b>&nbsp;<em>(please name all)</em></td>
                <td>
BSc Bioinformatics</p>BSc Biological Sciences</p>
</td>
            </tr>
        </table>


	<p>	<b>Mode of Delivery and Hours</b>	</p>	
		<table border="1" style="width: 600px; BORDER-COLLAPSE: collapse;">		
			<tr>
				<td bgcolor="#ADD8E6">&nbsp; &nbsp;</td>
				<td bgcolor="#ADD8E6"><b>Lectures</b></td>
				<td bgcolor="#ADD8E6"><b>Seminars</b></td>
				<td bgcolor="#ADD8E6"><b>Tutorials</b></td>
				<td bgcolor="#ADD8E6"><b>Lab / Practicals</b></td>
				<td bgcolor="#ADD8E6"><b>Fieldwork / Placement</b></td>
				<td bgcolor="#ADD8E6"><b>Other</b> <em>(Private study)</em></td>
				<td bgcolor="#ADD8E6"><b>Total</b></td>
			</tr>
			
			<tr>
				<td bgcolor="#ADD8E6"><b>Hours / Semester</b></td>
				<td>26</td>
				<td></td>
				<td>13</td>
				<td>11</td>
				<td></td>
				<td>100</td>
				<td>150</td>
			</tr>
			
			<tr>
				<td bgcolor="#ADD8E6"><b>Delivery Pattern</b></td>
				<td>13*2</td>
				<td></td>
				<td>13*1</td>
				<td>11*1</td>
				<td></td>
				<td></td>
				<td></td>
			</tr>					
		</table>
		

               	
         	<p>	  <b>Description</b> </p>
        <table border="1" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">
            <tr><td bgcolor="#ADD8E6"><b>Aims and Fit of Module</b><em>(i.e. relationship to programme)</em></td></tr>			
			<tr><td><pre>A major grand challenge in the molecular biosciences is to develop predictive models for key life processes. Accordingly, there is an increasing need for research bio-scientists to handle diverse, large-scale, quantitative datasets and to extract meaningful information from them.


This module is designed to provide the bioinformatics skills necessary to analyse and interpret genome sequence data, as well as high-throughput transcriptomic and proteomic data. The module will also provide an introduction to data integration and systems biology approaches
by emphasizing the system-related concept. The students should be able to process various datasets generated from different high-throughput technologies.
</pre></td> </tr>

			<tr><td bgcolor="#ADD8E6"><b>Learning Outcomes</b><em>(for accreditation and other reasons,sub-headings could be added to this section)<br/>Students completing the module successfully should be able to:</em></td></tr>		
			<tr><td><pre>A. Apply appropriate bioinformatics tools and statistical approaches to conduct a research project in this area with high-throughput biological data generated using latest biotechnology.
B. Critically analyze and process high-throughput bio-molecular data, especially next generation sequencing data, such as, RNA-Seq, ChIP-Seq, and SNPs.

C. Appraise, integrate and present different types of bioinformatics data in appropriate formats, such as, FASTA, FASTQ, BAM, and GTF.

D. Apply appropriate bioinformatics/statistical tools to analyze bioinformatics data on both Linux and Windows OS with R/Bioconductor.

E. Integrate different datasets from multiple layers to understand biology mechanisms from systems biology perspective.

F. Have a working knowledge of high performance computing systems, including bash scripting, multi-threads computing, and parallel computation.
</pre></td> </tr>

			<tr><td bgcolor="#ADD8E6"><b>Method of Teaching and Learning</b></td></tr>	
			<tr><td><pre>1.	Didactic component - the core of the teaching is lecture-based with Q/A and feedback.

2.	Self-learning component - students are encouraged to read around the subject materials.


3.	Comprehension/review exercise - two continuous assessments, following supervised discussion and Q/A sessions in the seminars.

4.	Case studies will be supplied to help students place the course material in context.

5.	Working in computer lab with bioinformatics tools to solve practical problems.
</pre></td></tr>

			<tr><td bgcolor="#ADD8E6"><b>Syllabus</b></td></tr>	
			<tr><td><pre>1.	Data analysis and visualization (weeks 1-3)

Use of open source platforms packages to visualize complex data types and integrate them
with diverse types of attribute data. This section will specifically cover R basics, various data visualization techniques, and advanced techniques, such as, ggplot2 package.


•	Lecture 1: R basics and data visualization techniques


•	Lecture 2: Advanced graphics with ggplot2


•	Lecture 3: Bioinformatics analysis with R/Bioconductor packages



2.	Next generation sequencing basics (weeks 4-6)


 
Library construction, strand-specific, various sequencing protocol (Single-End, Paired-End), recent development, file format, low level processing with Samtools, quality control, Linux Bash script


•	Lecture 4: Current development of next generation sequencing and third generation of sequencing.


•	Lecture 5: Linux Bash scripting for biological data processing.


•	Lecture 6: Raw NGS data quality control and secondary analysis.



3.	RNA-Seq data analysis (weeks 7-8)


Transcriptome  reconstruction  and  gene  expression  analysis  (differential  analysis,  isoform
analysis, visualization) with Tuxedo package (Tophat, Cufflinks, Cuffdiff and CummeRbund), work with real data. Get familiar with biological replicates, differential expression, visualization of NGS data, etc. Report and presentation.


•	Lecture 7: RNA-Seq and RNAseq data analysis methods


•	Lecture 8: RNA-Seq data analysis with Tuxedo package



4.	ChIP-Seq  data analysis (weeks 9-10)


Covers methods for analyzing other sequencing approaches, such as, ChIP-Seq, BS-Seq and
MeRIP-Seq; concept of consensus sequence (motif), and differential binding analysis. Reports and presentation.

•	Lecture 9: ChIP-Seq and other sequencing approaches, and their analysis methods


•	Lecture 10: ChIP-Seq analysis hands-on session, with MACS, MEME, etc.



5.	Proteomic and metabolomic data analyses (weeks 11-12)


Covers methods for proteomic and metabolomics data analysis with commercial and open-
source software tools


•	Lecture 11: Proteomics approach and  analysis methods


•	Lecture 12: Metabolomics approach and analysis methods





Additional state-of-the bioinformatics contents, especially systems biology, may be delivered depending on the actual progress of teaching & learning activities in the form of workshops, including personalized medicine, genome-based disease risk prediction (highly recommended), data integration, gene network prediction from high-throughput data, etc.
</pre></td> </tr>
        </table>	


			<p>	<b>Assessment</b></p>
		<em>Initial Assessment</em>
     <table border="1" style="width: 600px; text-align: center; BORDER-COLLAPSE: collapse;">
			<tr>
				<td bgcolor="#ADD8E6"><b>Sequence</b></td>
				<td bgcolor="#ADD8E6"><b>Method</b></td>
                                           <td bgcolor="#ADD8E6"><b>Assessment Type(EXAM or CW)<sup>2</sup></b></td>
				<td bgcolor="#ADD8E6"><b>Learning outcomes assessed</b><em>(use codes under Learning Outcomes)</em></td>	
				<td bgcolor="#ADD8E6"><b>Duration</b></td>
                                            <td bgcolor="#ADD8E6"><b>Week</b></td>				
				<td bgcolor="#ADD8E6"><b>% of Final Mark</b></td>
				<td bgcolor="#ADD8E6"><b>Resit(Y/N/S)<sup>3</sup></b></td>
			</tr>
					
            <tr>
				<td>001</td>
				<td>Project 1</td>
				<td>CW</td>
				<td>ABCDEF</td>
				<td></td>	
				<td></td>	
				<td>30</td>
                                             <td>N</td>
			</tr>	
					
            <tr>
				<td>002</td>
				<td>Project 2</td>
				<td>CW</td>
				<td>ABCDEF</td>
				<td></td>	
				<td></td>	
				<td>30</td>
                                             <td>N</td>
			</tr>	
					
            <tr>
				<td>003</td>
				<td>Project 3</td>
				<td>CW</td>
				<td>ABCDEF</td>
				<td></td>	
				<td></td>	
				<td>40</td>
                                             <td>N</td>
			</tr>	
			
	 </table>

	<em>Resit Assessment</em>
     <table border="1" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">
			<tr>
                <td bgcolor="#ADD8E6"><b>Sequence</b></td>
                <td bgcolor="#ADD8E6"><b>Assessment Type<br/>(EXAM or CW)</b></td>
				<td bgcolor="#ADD8E6"><b>Learning outcomes assessed</b><em><br/>(use codes under Learning Outcomes)</em></td>	
				<td bgcolor="#ADD8E6"><b>Duration</b></td>
                <td bgcolor="#ADD8E6"><b>Week</b></td>				
				<td bgcolor="#ADD8E6"><b>% of Final Mark</b></td>
			</tr>
			

			
	 </table>

		 <table border="0" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">

			<tr><td  class="style1">Reassessment opportunities in the same academic year are not available for final year undergraduate students. Any students eligible to be reassessed in a failed Year 4 (Level 3) module will have the resit opportunity at the next ordinary sitting of the examination/assessment</td></tr>

		 </table>
		

			<p>	<b>Textbooks</b><sup>4</sup></p>
                                <em>Mandatory Textbooks</em>
		<table border="1" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">
			<tr>
				<th style="width: 300px; text-align: left;" bgcolor="#ADD8E6">Title</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">Author</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">ISBN/Publisher</th>
			</tr>				
				
			 	
		</table>

                                <em>Optional Textbooks</em>
		<table border="1" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">
			<tr>
				<th style="width: 300px; text-align: left;" bgcolor="#ADD8E6">Title</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">Author</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">ISBN/Publisher</th>
			</tr>
			
			
			
		</table>

                                <em>Reference Textbooks</em>
		<table border="1" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse; ">
			<tr>
				<th style="width: 300px; text-align: left;" bgcolor="#ADD8E6">Title</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">Author</th>
				<th style="width: 150px; text-align: left;" bgcolor="#ADD8E6">ISBN/Publisher</th>
			</tr>
			
			
			

			<tr>
				<td style="font-size:13px;">BIOINFORMATICS AND FUNCTIONAL GENOMICS</td>
				<td style="font-size:13px;">JONATHAN PEVSNER</td>
				<td style="font-size:13px;">9780470085851 /&QUOT;JOHN WILEY & SONS INC&QUOT;,</td>
			</tr>
			

			<tr>
				<td style="font-size:13px;">UNDERSTANDING BIOINFORMATICS</td>
				<td style="font-size:13px;">ZVELEBIL, M., BAUM, J.O.</td>
				<td style="font-size:13px;">9780815340249 /GARLAND SCIENCE (2007)</td>
			</tr>
			

			<tr>
				<td style="font-size:13px;">BASH GUIDE FOR BEGINNERS</td>
				<td style="font-size:13px;">MACHTELT GARRELS</td>
				<td style="font-size:13px;"> GARRELS BVBA</td>
			</tr>
			

			<tr>
				<td style="font-size:13px;">A BEGINNER’S GUIDE TO R</td>
				<td style="font-size:13px;">ALAIN F. ZUUR L ELENA N. IENO L  ERIK  H.W.G.</td>
				<td style="font-size:13px;">9780387938363 /SPRINGER</td>
			</tr>
			

			<tr>
				<td style="font-size:13px;">BIOINFORMATICS WITH R COOKBOOK</td>
				<td style="font-size:13px;">PAURUSH PRAVEEN SINHA</td>
				<td style="font-size:13px;">9781783283132 /PACKT PUBLISHING LTD</td>
			</tr>
			
		</table>

<table border="0" style="width: 600px; text-align: left; BORDER-COLLAPSE: collapse;">
<tr><td align="left">&nbsp;</td></tr>
<tr><td align="left"><hr width="30%" align="left"></td></tr>
<tr>
<td class="style1"><sup>1</sup>Undergraduate programmes Year 1/Level 0 is equivalent to FHEQ Level 3, Year 2/Level 1 is equivalent to FHEQ Level 4, Year 3/Level 2 is equivalent to FHEQ Level 5, Year 4/Level 3 is equivalent to FHEQ Level 6, and Master programme Level 4 is equivalent to FHEQ Level 7.  FHEQ stands for “The Frameworks for HE Qualifications of UK Degree-Awarding Bodies”. It applies to degrees, diplomas, certificates and other academic awards (other than honorary degrees and higher doctorates) granted by a higher education provider in the exercise of its degree awarding powers. FHEQ is an important reference point for providers of higher education. These are numbered 4-8, succeeding levels 1-3 which precede higher education in “The National Qualifications Framework and The Qualifications and Credit Framework (NQF/QCF)”.  For more information please refer to: <a href='https://www.qaa.ac.uk/quality-code/qualifications-and-credit-frameworks'>https://www.qaa.ac.uk/quality-code/qualifications-and-credit-frameworks</a>.</td>
</tr>
<tr>
<td class="style1"><sup>2</sup>Examination (EXAM) is an assessment task formally scheduled and supervised by the University which takes place over a specified period, in a specified location and at a specified time, e.g. mid-term, final, resit etc. Coursework or continuously assessed work (CW) is different than EXAM; it is undertaken throughout the semester organized by the department.</td>
</tr>
<tr>
<td class="style1"><sup>3</sup>“Y” means that the corresponding resit is an independent assessment covering the same learning outcomes, in the same assessment type and bearing the same weighting with the original assessment; “N” means there is no resit opportunity for the particular component; “S” means there will be a single resit that will be designed to assess all learning outcomes of the module, and will be weighted as 100% of the final module mark, disregarding any marks achieved at the original assessment.</td>
</tr>
<td class="style1"><sup>4</sup>Mandatory: every student will be charged for a copy of the text, whether or not they collect it<br>
Optional: the Library will purchase a number of copies for students to purchase if they wish<br>
Reference: the Library will add a couple of copies to their collection<br>
Textbooks are subject to change until the final procurement complete.
</td>
</tr>
</table>


<br>

</body>

</html>